### ISSUE TITLE
Proxo detects the universal bypass extension

### ISSUE THREAD

**Issue by wernser412** [Sainan/Universal-Bypass#1132]
The "Proxo" program detects the universal bypass extension 

Link Program: https://wearedevs.net/d/Proxo

![Sin título](https://user-images.githubusercontent.com/53667212/78979094-bec3fe00-7ae0-11ea-8e8c-30fbfa028015.png)

**Comment by timmyRS** [comment]
Oh no, this virus detects that I'm using Avast!

Don't use the software, don't use Universal Bypass, or don't use Chrome. You decide.

**Comment by MIHAIdavid22** [comment]
tl;dr just run the universal bypass extention unpacked. easy fix.

i've dealt with this problem before,there is a pretty easy way to fix it. you must simply install universal bypass unloaded instead of the usual chrome download.the program checks for the id of universal bypass inside your chrome's extensions .to fix this you must go to appdata by using the run program (win + r key at the same time) then navigate Local->Google->Chrome->User Data->Default->Extensions there you will see a random file called aihomhdbhpnpmcnnbckjjcebjoikpihj move that file to the desktop or downloads (just so its easier for the next steps) then remove universal bypass using remove from chrome button and then click the 3 dots then more tools then extentions and after that in the top right corner press developer mode and click load unpacked.After that you want to select the folder inside aihomhdbhpnpmcnnbckjjcebjoikpihj (the numbers folder) and click Select Folder.after that you are pretty much done.the only downside is,you will have to re-download and re-do the steps for the extention every time theres an update,its not necesary to update but it would be recommended.

**Comment by timmyRS** [comment]
Google isn't the fastest to get an update out, but once a version is available on Chrome, I do assume that everyone has it, so even though you will get bypass definitions, they might not work perfectly for your old version anymore. :)

**Comment by MIHAIdavid22** [comment]
i guess thats true but updating it is very simple and easy and allows for universal bypass to not get detected by some programs or certain sites.

**Comment by ghajini** [comment]
> Oh no, this virus detects that I'm using Avast!
> 
> Don't use the software, don't use Universal Bypass, or don't use Chrome. You decide.

Report to avast at least they gonna flag this as malicious 😅
